app/javascript/modules/blacklight-override.js

```diff
@@ -0,0 +1,15 @@
+// Override Blacklights default error handler to give a more useful error message
+// Ported upstream as https://github.com/projectblacklight/blacklight/pull/2386
+Blacklight.modal.onFailure = function(jqXHR, textStatus, errorThrown) {
```

Comment:
what is textStatus?  Is it helpful in any way?

Reply:
It is the variable jquery uses for the status: https://api.jquery.com/jquery.ajax/#jqXHR

Reply:
thanks.